docs/management/api/scan-api.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[scan-api]]
+=== Scan a file or folder
+
+Scan a file or folder on a host running {elastic-defend} for malware.
```

Comment:
The end bit "for malware" sounds like the host must be running Defend AND have "Malware protections" enabled, but I'm not sure that's the case? Maybe it means we're scanning for malware:
```suggestion
Scan a file or folder for malware on a host running {elastic-defend}.
```